docs/area-owners.md

```diff
@@ -12,25 +12,24 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
+| area-Codegen-Interpreter-mono                  | @vitek-karas         | @BrzVlad @kotlarmilos                               |                                                                                                                                                                                                                                                                                           |
+| area-Codegen-Intrinsics-mono                   | @steveisok           | @fanyang-mono                                       |                                                                                                                                                                                                                                                                                           |
+| area-Codegen-JIT-mono                          | @steveisok           | @vargaz                                             |                                                                                                                                                                                                                                                                                           |
+| area-Codegen-LLVM-mono                         | @steveisok           | @vargaz                                             |                                                                                                                                                                                                                                                                                           |
+| area-Codegen-meta-mono                         | @steveisok           | @vargaz                                             |                                                                                                                                                                                                                                                                                           |
+| area-CrossGen/NGEN-coreclr                     | @steveisok           | @dotnet/crossgen-contrib                            |                                                                                                                                                                                                                                                                                           |
+| area-crossgen2-coreclr                         | @steveisok           | @dotnet/crossgen-contrib                            |                                                                                                                                                                                                                                                                                           |
```

Comment:
@steveisok do we need to add Fan, Zoltan and Kate to https://github.com/orgs/dotnet/teams/crossgen-contrib ?